### PR TITLE
Add the missing section in the document about AccessLogWriter

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: site/src/sphinx/server-access-log.rst [hunk 1]
```diff
@@ -104,6 +107,28 @@ using :api:`ServerBuilder`. You may use one of the pre-defined log formats.
...
+      .service((ctx, req) -> HttpResponse.of(OK)); // This service will log using the combined format.
+      ...
+
+    sb.route()
+      .get("/commonAccessLog")
+      .accessLogWriter(AccessLogWriter.common(), true)
+      .build((ctx, req) -> HttpResponse.of(OK)); // This service will log using the common format.
+
+    AccessLogWriter myWriter = ...
+    sb.accessLogWriter(myWriter);
```
Comment (trustin) [inline site/src/sphinx/server-access-log.rst]: - How about renaming `myWriter` to `fallbackLogWriter`?
- How about moving the two sentences to the beginning of the example?

Comment (minwoox) [reply site/src/sphinx/server-access-log.rst]: Thanks! fixed! :bowing_man:

**Comment by trustin** [comment]
Thanks, @minwoox !

**Comment by minwoox** [comment]
Thanks for reviewing!

**Pull request merged**